keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentSummary.kt

```diff
@@ -26,12 +33,13 @@ data class EnvironmentSummary(
     artifacts.find { it.name == artifact.name && it.type == artifact.type }
       ?.let {
         when (version) {
-          it.versions.current -> PromotionStatus.CURRENT
-          it.versions.deploying -> PromotionStatus.DEPLOYING
-          in it.versions.previous -> PromotionStatus.PREVIOUS
-          in it.versions.approved -> PromotionStatus.APPROVED
-          in it.versions.pending -> PromotionStatus.PENDING
-          in it.versions.vetoed -> PromotionStatus.VETOED
+          it.versions.current -> CURRENT
```

Comment:
I've now configured IntelliJ to do this for me automatically. Didn't know it was a thing, sorry. 😄

Reply:
I got it from fletch, I didn't know it was a thing before him either. You'll have to show me how to make intellij do that automatically